### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by FlappyDev-ctrl** [lpedraoss/tetrio-agent#8]
I have a problems with ur bot you can send me your discord to add you ?

**Comment by brosgor** [comment]
Describe here the error you're facing or the confusion you have

**Comment by FlappyDev-ctrl** [comment]
![image](https://github.com/user-attachments/assets/b402018a-1747-4f40-b2be-ad90ebbd7eb4)
This

**Comment by brosgor** [comment]
### How to Use the Tetris Bot

To use the bot correctly, you'll need to capture the pixel coordinates where the Tetris pieces are located. Follow these steps:

#### 1. Capture the Pixel Coordinates
- Move your mouse to the exact location where the Tetris piece appears on the screen.
- Run the provided Python code to get the current mouse position and the color of the pixel at that position. The code will print the coordinates and the pixel color in the terminal.
- Note down these coordinates as they correspond to where the Tetris pieces will appear.

    ```python
    import pyautogui
    import pyscreeze

    # Get the current mouse cursor position
    x, y = pyautogui.position()

    # Get the color of the pixel at the cursor position
    pixel_color = pyscreeze.pixel(x, y)

    print(f"The cursor position is: ({x, y})")
    print(f"The pixel color at the cursor position is: {pixel_color}")
    ```

#### 2. Record the Pixel Coordinates
- After capturing the coordinates, store them in the `pixels` dictionary. This dictionary will map the pixel IDs to their corresponding coordinates. Here’s an example:

    ```python
    pixels = {
        # Example coordinates
        1: (937, 157),
        2: (946, 257),
        3: (952, 355), 
        4: (943, 451),
        5: (945, 544)
    }
    ```

#### 3. Execute the Bot
- Once you’ve set up the pixel dictionary, you can run the bot using the command `python main.py gui`. Keep in mind that this version of the bot is limited to 40 lines of code, so it’s still a bit basic and might need further refinement.

---

To summarize, capture the pixel positions of where the Tetris pieces will appear, record those positions in the `pixels` dictionary, and run the bot. This bot is currently in its early stages, so there’s room for improvement, but it should help you get started.

**Comment by FlappyDev-ctrl** [comment]
I managed to capture the position but where should I put the results obtained in the terminal?

What is the name of the .py where I have to enter the results?

**Comment by FlappyDev-ctrl** [comment]
You can make video to do that pls bcs i can't ,-, or add flqppy on discord to explain on screenshare pls

**Comment by Meekiavelique** [comment]
Even after doing what what you told, the bot isn't detecting the game well 

Showing this :
```..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
..........
#.....####
###..####.
```

Even if the game look like this
![image](https://github.com/user-attachments/assets/c6ab7462-38f2-4a05-9831-dddb133cbb15)

**Comment by brosgor** [comment]
![example(1)(3)](https://github.com/user-attachments/assets/00e18d8d-687c-4e27-abad-7f2d2d5f0017)
The issue lies in pixel capture. Next week, I'll work on making pixel capture easier to avoid these kinds of problems.

**Comment by brosgor** [comment]
> You can make video to do that pls bcs i can't ,-, or add flqppy on discord to explain on screenshare pls

### Update on Pixel Capture Enhancement

I’ve pushed changes to the `development` branch that should address the pixel capture issues. 

- **Proportional Pixel Coordinates**: The pixel positions now automatically adjust according to different screen resolutions. This means you won't need to manually change pixel coordinates for different setups anymore.
- **Screen Resolution Adaptation**: The code now calculates proportional coordinates based on the original capture resolution, ensuring consistent performance across various configurations.

To test the changes, please pull the latest code from the `development` branch and run the following steps:
1. Execute `python main.py gui`.
2. Click on `Load`.
3. Then, click on `Start`.

This should make the bot functional without the need for further adjustments. Let me know how it goes!

**Comment by Meekiavelique** [comment]
Thanks
